[
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/54024070",
    "html_url": "https://github.com/electron/electron/issues/620#issuecomment-54024070",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/620",
    "id": 54024070,
    "user": {
      "login": "zcbenz",
      "id": 639601,
      "avatar_url": "https://avatars3.githubusercontent.com/u/639601?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/zcbenz",
      "html_url": "https://github.com/zcbenz",
      "followers_url": "https://api.github.com/users/zcbenz/followers",
      "following_url": "https://api.github.com/users/zcbenz/following{/other_user}",
      "gists_url": "https://api.github.com/users/zcbenz/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/zcbenz/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/zcbenz/subscriptions",
      "organizations_url": "https://api.github.com/users/zcbenz/orgs",
      "repos_url": "https://api.github.com/users/zcbenz/repos",
      "events_url": "https://api.github.com/users/zcbenz/events{/privacy}",
      "received_events_url": "https://api.github.com/users/zcbenz/received_events",
      "type": "User",
      "site_admin": true
    },
    "created_at": "2014-09-01T06:42:10Z",
    "updated_at": "2014-09-01T06:42:10Z",
    "body": "The error said you didn't have access to `/Users/kelvn/.atom/.node-gyp/.node-gyp/0.11.13`, I think you can try removing the `/Users/kelvn/.atom/.node-gyp/` and run again.\n"
  }
]